cmd/controller/sidecarexec.go

```diff
@@ -13,23 +16,48 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
-func sidecarexecMain() {
+func sidecarexecMain(debug bool, debugArgs []string) {
+	localCmdRunner := exec.NewPlainCmdRunner()
+	sandboxCmdRunner := sidecarexec.NewSandboxCmdRunner(localCmdRunner, sidecarexec.SandboxCmdRunnerOpts{
+		RequiresPosix: map[string]bool{
+			"vendir": true,
+			"bash":   true, // for debugging
+		},
+		RequiresNetwork: map[string]bool{
+			"vendir": true,
+			"kbld":   true,
+		},
+	})
+
+	if debug {
```

Comment:
it looks like the use-case of this debug mode is "I want to see the output of a command run in the side-car" ?     can you also add that to someplace like
https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/docs/dev.md#troubleshooting-tips

and/or the docs/website repo?  (eg. https://carvel.dev/kapp-controller/docs/v0.38.0/debugging-kc/ )

Reply:
yup, this would be a good addition to the docs. the intent was to allow to debug command execution in a same way that kapp-controller is executing it.